README.md

```diff
@@ -50,6 +50,30 @@ add the following parameter to the close function:
 ```javascript
     worker.close(false);
 ```
+
+### Events
+
+To organize your own tests, the worker instance has an `emitter` property
+which allows you to listen to several events with the
+[EventEmitter API](https://nodejs.org/api/events.html#events_class_eventemitter).
+
+Available events:
+
+- `task.completed`: emitted when a task has been completed.
+- `task.retried`: emitted when a task is going to be retried because the
+ handler has failed.
+- `task.failed`: emitted when a task has failed because the handler
+ has failed on a task which was already being retried.
+- `worker.closed`: emitted when the worker has been closed.
```

Comment:
yes, but I think it's still ok. A lot of methods will have their contract violated by a process.exit(). Actually I don't think there is a way to avoid this.
The more I think about it the more I think it's ugly that a library kills the process... but fixing this is not in the scope of this PR.